wire-compiler/src/main/java/com/squareup/wire/schema/NewSchemaLoader.kt

```diff
@@ -16,10 +16,13 @@
 package com.squareup.wire.schema
 
 import com.google.common.io.Closer
+import com.squareup.wire.java.Profile
+import com.squareup.wire.java.internal.ProfileFileElement
```

Comment:
nit: should these types be moved out of the `internal` subpackage?

Reply:
@swankjesse  has plans for that we, we will follow up